controllers/controller.php

```diff
@@ -66,6 +66,7 @@ function __construct($f3)
         }
 
         $this->_f3 = $f3;
```

Comment:
This line of code needs to be resolved as this results into a conflict. In the master branch, we have set this at the beginning of the constructor. In the site color branch, it's declared at a different instance.